### PR TITLE
Add org.gnome.World.Citations

### PR DESCRIPTION
**Pull request by A6GibKm**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

---

This is my new app to manage bibtex files, The icon is temporal, this is a preview release.

### CODE REVIEW

**Comment by A6GibKm** [comment]
bot, build

**Comment by A6GibKm** [comment]
bot, build org.gnome.World.Citations

**Comment by hfiguiere** [comment]
it needs a non-release adwaita.

File: org.gnome.World.Citations.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+    "app-id": "org.gnome.World.Citations",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "42",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+    "command": "citations",
+    "finish-args" : [
+        "--socket=fallback-x11",
```
Comment (hfiguiere) [inline org.gnome.World.Citations.json:9]: --share=ipc is still needed for X11

**Comment by A6GibKm** [comment]
> it needs a non-release adwaita.

Yeah forgot to bundle it  is this bad? The features I need wont be released till gnome 43 alpha.

**Comment by hfiguiere** [comment]
If it work with that specific release, it's fine, but the point was just to solve the build failure :-)

File: org.gnome.World.Citations.json (hunk 1 above)
Comment (A6GibKm) [reply org.gnome.World.Citations.json:9]: Adding it, but what does it do anyways?

**Comment by A6GibKm** [comment]
bot, build org.gnome.World.Citations

File: org.gnome.World.Citations.json (hunk 1 above)
Comment (hfiguiere) [reply org.gnome.World.Citations.json:11]: x11 needs it. without that it's slower

Comment (A6GibKm) [reply org.gnome.World.Citations.json:11]: But like generally slower in general?

**Pull request closed without merging**